mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -1,9 +1,17 @@
 CLASS net/minecraft/class_77 net/minecraft/client/particle/Particle
+	FIELD field_2635 textureId I
+	FIELD field_2636 prevU F
+	FIELD field_2637 prevV F
+	FIELD field_2638 particleAge I
 	FIELD field_2639 maxAge I
 	FIELD field_2640 scale F
+	FIELD field_2641 gravity F
```

Comment:
[gravityStrength](https://maven.fabricmc.net/docs/yarn-1.21.1+build.3/net/minecraft/client/particle/Particle.html#gravityStrength) in Yarn

Reply:
forgor this one gg

Reply:
done